native-messaging/app/ping_pong.py

```diff
@@ -18,7 +18,13 @@ def getMessage():
     # Encode a message for transmission,
     # given its content.
     def encodeMessage(messageContent):
-        encodedContent = json.dumps(messageContent).encode('utf-8')
+        # https://github.com/mdn/webextensions-examples/issues/509
+        # https://discuss.python.org/t/how-to-read-1mb-of-input-from-stdin/22534/19
+        # https://stackoverflow.com/a/56563264
```

Comment:
Let's remove these three lines. The link to the Python documentation below is sufficient. Your GH issue and Python code would be causing more confusion than that it would help, because these all frame the issue as "issue with reading 1MB", while the actual issue is with writing 1 MB from the Python script (or rather, the browser not accepting 1 MB).